README.md

```diff
@@ -191,3 +191,8 @@ If your application depends on a package that client-go depends on, and you let
 
 ### Contributing code
 Please send pull requests against the client packages in the Kubernetes main [repository](https://github.com/kubernetes/kubernetes). Changes in the staging area will be published to this repository every day.
+
+
+## License
+
+This program follows the Apache License version 2.0 (http://www.apache.org/licenses/ ).  See LICENSE included with the distribution for details.
```

Comment:
@berroteran Thanks for your PR! However, PRs to client-go need to go to the main repository i.e. kubernetes/kubernetes. More details are in the contributing section in the README.

Having said that, I don't think we need to include this line here. https://github.com/kubernetes/kubernetes-template-project defines the content that all READMEs that all kubernetes repos need to have and it doesn't include it.

/close